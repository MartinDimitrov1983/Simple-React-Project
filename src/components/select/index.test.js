import React from 'react'
import { render } from '@testing-library/react'

import Select from './index'

test('Snapshot Select', () => {
    const { asFragment } = render(<Select />)
    expect(asFragment()).toMatchSnapshot()
})
