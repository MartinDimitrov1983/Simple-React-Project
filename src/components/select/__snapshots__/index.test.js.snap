// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Select 1`] = `
<DocumentFragment>
  <select
    class="select"
  >
    <option
      value="none"
    >
      Choose Region
    </option>
  </select>
</DocumentFragment>
`;
